File: frontend/routes/activity.route.js
const router = require("express").Router();

const {
  middlewareCheckGoogleProfile,
  middlewareCheckEmail,
} = require('../controllers/oauth-google.controller');

const {
  getRoomActivity,
  deleteRoomActivity,
} = require('../controllers/activity.controller');

router.get(/\/([A-Za-z0-9]+)/, middlewareCheckGoogleProfile, middlewareCheckEmail, getRoomActivity);
router.delete(/\/([A-Za-z0-9]+)\/deleteChat/, middlewareCheckGoogleProfile, middlewareCheckEmail, deleteRoomActivity);

module.exports = router;

File: backend/routes/activity.route.go
package routes

import (
	"github.com/gofiber/fiber/v2"
	controller "github.com/simopunkc/chirpbird-v2/controllers"
)

func ActivityRoute(app fiber.Router) {
	app.Get("/:activity", controller.GetRoomActivity)
	app.Delete("/:activity/deleteChat", controller.DeleteRoomActivity)
}
